### PR TITLE
User service created

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

File: src/main/java/mate/academy/exception/AuthenticationException.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends Exception {
+    public AuthenticationException(String message) {
+        super(message);
+    }
+
+    public AuthenticationException(String message, Throwable cause) {
+        super(message, cause);
+    }
```
Comment (nazariipertsak) [inline src/main/java/mate/academy/exception/AuthenticationException.java:10]: Why you need this constructor? You don't use it

File: src/main/java/mate/academy/exception/RegistrationException.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class RegistrationException extends Exception {
+    public RegistrationException(String message) {
+        super(message);
+    }
+
+    public RegistrationException(String message, Throwable cause) {
+        super(message, cause);
+    }
```
Comment (nazariipertsak) [inline src/main/java/mate/academy/exception/RegistrationException.java:10]: Same here

File: src/main/resources/hibernate.cfg.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        <property name="hibernate.connection.password">1234root</property>
+        <property name="hbm2ddl.auto">update</property>
+
+        <!-- List of XML mapping files -->
+        <mapping class="mate.academy.model.MovieSession"/>
+        <mapping class="mate.academy.model.Movie"/>
+        <mapping class="mate.academy.model.CinemaHall"/>
+        <mapping class="mate.academy.model.User"/>
+    </session-factory>
+</hibernate-configuration>
```
Comment (nazariipertsak) [inline src/main/resources/hibernate.cfg.xml:16]: No newline at end of file :)

**Review by nazariipertsak — CHANGES_REQUESTED** [review]

**Review by nazariipertsak — APPROVED** [review]
Great!

File: src/main/java/mate/academy/util/HashUtil.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public static String hashPassword(String password, byte[] salt) {
+        StringBuilder hashedPassword = new StringBuilder();
+        try {
+            MessageDigest messageDigest = MessageDigest.getInstance(CRYPTO_ALGORITHM);
+            messageDigest.update(salt);
+            byte[] digest = messageDigest.digest(password.getBytes());
+            for (byte b : digest) {
+                hashedPassword.append(String.format("%02x", b));
```
Comment (ChumakAn) [inline src/main/java/mate/academy/util/HashUtil.java:28]: can be a constant too.

**Review by ChumakAn — CHANGES_REQUESTED** [review]
Good! Minor comment

**Review by ChumakAn — APPROVED** [review]

**Pull request closed without merging**